mitodl/ocw-studio
# update ocw import to match new ocw-to-hugo output structure

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-to-hugo/issues/214

#### What's this PR do?
This PR alters the way `ocw-studio` parses `ocw-to-hugo` output to match the new output structure as detailed in [this](https://github.com/mitodl/ocw-to-hugo/pull/216) PR.  Some issues were also corrected with the way the import script determines the UID of a page.  Rather than trying to grab the UID from the menu, it's now taken from the `uid` property on the front matter of any given page.  The "page or resource" question is answered by checking the `layout` property and comparing it to known layouts for each type.  Root level pages also now correctly set the course home page as their parent.

#### How should this be manually tested?
I'm not exactly sure how this could be manually tested aside from the unit tests, as the import script is set up to pull directly from an s3 bucket and cannot be redirected at a local folder.  Either way, the unit tests should be checked locally against the new test data.
